lib/rules/no-plusplus.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @fileoverview add fixer to rule no-plusplus
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+"use strict";
+
+const ruleComposer = require("eslint-rule-composer");
+const utils = require("../utils");
+
+const rule = utils.getFixableRule("no-plusplus");
+const fixableTypes = new Set(["ExpressionStatement", "ForStatement"]);
+
+module.exports = ruleComposer.mapReports(
+    rule,
+    problem => {
+        const node = problem.node;
+        const op = node.operator === "++" ? "+=1" : "-=1";
+
+        problem.fix = function(fixer) {
```

Comment:
Non-blocking: we can use arrow function.

Reply:
[image](https://user-images.githubusercontent.com/13050025/52550628-cff97800-2e13-11e9-82b3-b35f07eb29d6.png)

we got an error here, but it seems a bug to me.(I've add a paren on the expression.)

Reply:
It's defined at `eslint-config-eslint`, not a parsing error. Use `function` back or disable that rule with a comment?

Reply:
IMO, the rule should not report in this case, you think?

Reply:
Add `allowParens` option for this rule in global ESLint config.

Reply:
the option might need to be enabled by default, can you file an issue in eslint repo?

Reply:
Hmm...I think they may have their opinion, so I'm not going to do that.